float64[] Joints
float64[] Pos
float64[] EulerZYX